src/Controller/LegaleController.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+namespace App\Controller;
+
+class LegaleController extends AbstractController
+{
+    public function legale(): string
```

Comment:
index method for better coherence

Reply:
do you mean replace "legale" by "index" inside the method ?